distribution/rcp-product/org.wso2.integrationstudio.rcp.product/integrationstudio.product

```diff
@@ -52,6 +52,7 @@ Please visit : http://wso2.com/products/integration-studio/
       </win>
    </launcher>
 
+
```

Comment:
Do we need this new line?

Reply:
Fixing : [1129](https://github.com/wso2/integration-studio/pull/1129)